docs/ui/workspaces.md

```diff
@@ -97,4 +97,36 @@ Select **Send** to initiate the invitation.
 
 To add a service account to a workspace, select the Service Accounts **+** icon. You can select from a list of existing service accounts configured for the organization. Select a Workspace Role for the service account. This will be the initial role for the service account within the workspace. A workspace Owner can change this role at any time. Select **Share** to finalize adding the service account.
 
-To delete a workspace collaborator or service account, select **Remove** from the menu on the right side of the user or service account information on this page.
\ No newline at end of file
+To delete a workspace collaborator or service account, select **Remove** from the menu on the right side of the user or service account information on this page.
+
+## Workspace transfer
+
+Workspace transfer enables you to move an existing workspace from one account to another. For example, you may transfer a workspace from a personal account to an organization.
+
+Transferring a workspace enables you to take existing configuration and flow run history with you as projects move to different teams or organizations. Without transfer, users moving to a new organization from, for example, a personal account would need to re-create all of their block, deployment, notification, and work queue configuration in a new workspace. Additionally, any flow run history would remain in the personal workspace. 
+
+!!! note "Workspace transfer permissions"
+    Workspace transfer must be initiated or approved by a user with admin priviliges for the workspace to be transferred.
+
+    For example, if you are transferring a personal workspace to an organization, the owner of the personal account is the default admin for that account and must initiate or approve the transfer.
```

Comment:
It may be useful to also call out here that to initiate a personal->personal workspace transfer, users should reach out to support@prefect.io